tensorflow/core/common_runtime/eager/execute.cc

```diff
@@ -1294,12 +1294,12 @@ Status AddOrExecuteNode(core::RefCountPtr<KernelAndDevice> kernel,
   const int num_outputs = kernel->num_outputs();
   absl::optional<EagerFunctionParams> eager_func_params =
       op->eager_func_params();
+  // Create an eager op id for a cross-process function if not exist.
   if (kernel->IsCrossProcess() && !eager_func_params.has_value()) {
-    // Create an eager op id for a cross-process function if not exist.
 #if defined(IS_MOBILE_PLATFORM)
     return errors::Unimplemented(
         "Cross-process functions are not supported on mobile devices.");
-#else  // !IS_MOBILE_PLATFORM
+#else   // !IS_MOBILE_PLATFORM
```

Comment:
Nit: There are two more places
```suggestion
#else  // !IS_MOBILE_PLATFORM
```

Reply:
done